### PR TITLE
chore: Update ios publication :rocket:

### PR DESCRIPTION
**Pull request by kosssi**
- [x] Fix order parameter
- [x] Add plugin to respond automatically to Apple question: It's app uses non exempt encryption?
- [x] Fix cozy team name
- [x] Add npm command to publish ios

### CODE REVIEW

File: mobile/config.xml [hunk 1]
```diff
@@ -62,4 +59,5 @@
     </plugin>
     <plugin name="cordova-plugin-network-information" spec="1.3.2" />
     <plugin name="cordova-plugin-background-fetch" spec="4.0.0" />
+    <plugin name="cordova-plugin-ios-no-export-compliance" spec="https://github.com/mikaoelitiana/cordova-plugin-ios-no-export-compliance.git" />
```
Comment (enguerran) [inline mobile/config.xml]: What's the aim of this plugin?

File: mobile/config.xml [hunk 2]
```diff
@@ -29,9 +29,6 @@
         <preference name="StatusBarStyle" value="default" />
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
-        <config-file mode="add" parent="ITSAppUsesNonExemptEncryption" target="*-Info.plist">
-            <false />
-        </config-file>
```
Comment (enguerran) [inline mobile/config.xml]: I don't know why you remove this, I don't know fastlane enough to understand.

File: package.json [hunk 3]
```diff
@@ -34,7 +34,8 @@
     "android:signed": "npm run android:release && (cd mobile && apksigner sign --ks keys/android/cozy-drive-release-key.jks --out build/android/cozy-drive.apk platforms/android/build/outputs/apk/android-release-unsigned.apk)",
     "android:publish": "npm run android:signed && (cd mobile && fastlane supply)",
     "ios:run": "(cd mobile && cordova run ios --device)",
-    "ios:run:emulator": "(cd mobile && cordova run ios --emulator)"
+    "ios:run:emulator": "(cd mobile && cordova run ios --emulator)",
+    "ios:publish": "(cd mobile && fastlane ios pushtest)"
```
Comment (enguerran) [inline package.json]: Ok 👍

File: mobile/config.xml (hunk 2 above)
Comment (kosssi) [inline mobile/config.xml]: When you publish a new version on Apple Store, they are a question:  It's app uses encryption?

Normally this code respond to false but it's never worked... so I decide to find another solution. With cordova-plugin-ios-no-export-compliance it's now ok 👍

Comment (enguerran) [inline mobile/config.xml]: If you say so, I trust you.

**Comment by enguerran** [comment]
@kosssi I am not able to approve (or disapprove) that Pull Request. But as we discussed about, and there is not much changes included, I let you merge if you think it's ok.

**Comment by kosssi** [comment]
👍

**Pull request merged**